lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -636,8 +622,13 @@ MITIGATION_02_CRI_DCMAW:
     criId: dcmaw
   parent: CRI_STATE
   events:
+    next:
+      targetState: POST_DCMAW_SUCCESS_PAGE_J5
     not-found:
-      targetState: PYI_NO_MATCH
+      targetState: MITIGATION_02_PYI_POST_OFFICE
+      checkIfDisabled:
+        f2f:
+          targetState: PYI_ANOTHER_WAY
     access-denied:
       targetState: MITIGATION_02_PYI_POST_OFFICE
```

Comment:
Be tempted to rename this to just `PYI_POST_OFFICE` given this is being shared by the KBV dropout and V03 mitigation routes now
And the parent state itself - maybe `CRI_DCMAW_PYI_ESCAPE`?